install.yaml

```diff
@@ -1,30 +1,31 @@
 name: xhgui
 
-ddev_version_constraint: ">=v1.24.0"
+# This add-on is a part of DDEV since v1.24.4
+ddev_version_constraint: ">=1.24.0-0, <1.24.4-0"
```

Comment:
I didn't know about the -0 technique. I used to use -alpha1 for odd comparisons. I like -0.

Reply:
We add `-0` in DDEV, but we don't check for such complex constraints like this, so I had to add it myself.

https://github.com/ddev/ddev/blob/main/pkg/ddevapp/utils.go#L554